easy-to-use-api/klips-chart-api/src/components/Chart/index.ts

```diff
@@ -80,10 +81,23 @@ export class ChartAPI {
 
     if (!chartDom) {
       throw new Error('No div found for chart rendering.');
-    }
+    };
 
     this.chart = echarts.init(chartDom as HTMLElement);
 
+    // select band to display
+    let lineTypeCompare: string = "solid";
+    if (params.band?.includes('perceived')) {
+      [legendSelect[0], legendSelect[1], legendSelect[2]] = [legendSelect[0], legendSelect[1], legendSelect[2]];
```

Comment:
This looks a bit strange to me. What you try is to change the order of the array, but I don't see where you are are using this array since it is not bound to a variable. Also applies for line 93, 95 and 97.

Maybe I don't get it. If this is the case please ignore and sorry for the noise.

Reply:
This changes the order of the legendSelect-Array (imported in line 59), which controls which series is shown first via the chart legend. This is done by creating a new array with the default order and asigning it to a new array with the new order.
(Line 91 is not strictly  necessairy, since band=perceived ist the default.)
It worked when I tested ist, but maybe there is a better way to do this? I'm open for suggestions.

Reply:
Thanks for the explanation. But then something like 

`legendSelect = [legendSelect[1], legendSelect[0], legendSelect[2]];`

would be enough, right?

Reply:
Maybe, however, legendSelect is an imported variable (declared in util/Chart.ts) because it is part of the baseChart configuration. So that way I would have to re-export an imported variable?

Reply:
Okay, if that's how this charting library works, there's nothing else to stop merging it from my point of view.

Reply:
Right, also OK from my side